apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -6225,13 +6216,14 @@ table td[class*=col-]:last-child {
     font-weight: 400;
 }
 
-.pd-r25 {
+.pdR25imp {
```

Comment:
This change is reverted while according to the code design, we've fixed most of the camel case classes. @prateek-giddh

Reply:
Done